l2geth/consensus/consensus.go

```diff
@@ -59,6 +59,9 @@ type Engine interface {
 	// TODO Add comment
 	SetBatchPeriod(bps *types.BatchPeriodStartMsg)
 
+	// returns the current scheduler address
+	GetSchedulerAddress() (common.Address, error)
```

Comment:
@curryxbo What is the best/correct way to get the address/public key of the scheduler blockchain validation/ApplyTransaction?

Reply:
I see in this PR (https://github.com/mantlenetworkio/mantle/pull/407/files) the scheduler address is available in l2geth/consensus/clique/scheduler.go Scheduler struct.

Reply:
@aodhgan Due to some changes to the schema, the scheduler unified production blocks, so this signature is now unnecessary，so the Issue will be closed